docs/source/endpoints/relations.md

```diff
@@ -0,0 +1,300 @@
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_post_with_uid.resp
+:language: http
+```
+
+**Failure**:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_post_failure.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_post_failure.resp
+:language: http
+```
+
+(restapi-relations-deleting-relations-label)=
+
+## Deleting relations
+
+Relations can be deleted by relation name, source object, target object, or a combination of these.
+Relations can also be deleted by providing a list of single relations.
+
+If a deleted relation is based on a `RelationChoice` or `RelationList` field, the value of the field is removed or updated accordingly on the source object.
+
+### Delete a list of single relations
+
+by **path**:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_path.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_path.resp
+:language: http
+```
+
+by **UID**:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_uid.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_uid.resp
+:language: http
+```
+
+**Failures** are listed in response:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_failure.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_failure.resp
+:language: http
+```
+
+### Delete relations by relation name
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_relationname.req
+```
+
+<!-- Uncomment with https://github.com/plone/plone.api/pull/502 merged
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_relationname.resp
+:language: http
+``` -->
+
+### Delete relations by source
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_source.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_source.resp
+:language: http
+```
+
+Delete relations by source UID or source path.
+
+### Delete relations by target
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/relations_del_target.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/relations_del_target.resp
+:language: http
+```
+
+Delete relations by target UID or target path.
+
+### Delete relations by combination of source/target and relation name
```

Comment:
```suggestion
### Delete relations by relation name and either a source or target
```

Reply:
In fact it's possible to delete by any combination of two or even all three of source, target and relation name.
Maybe you can come up with a proper wording?

And I think one example like the existing is enough. But I'm open for suggestions.